MCprep_addon/spawner/mcmodel.py

```diff
@@ -116,11 +116,14 @@ def add_element(
 	return verts, edges, faces
 
 
-def add_material(
+def add_get_material(
```

Comment:
Can you double-check that all instances of `add_material` have been changed to `add_get_material`?

Reply:
The only place it is used is model spawn add_model() so.